metadataStore/odm_templates.py

```diff
@@ -113,24 +146,23 @@ class Event(Document):
 
     Attributes
     ----------
-    descriptor : metadataStore.odm_templates.EventDescriptor
-        Foreign key to EventDescriptor that provides info regarding a
-        set of events of the same type
-
-    data : dict
-        Data dictionary where experimental data is stored.
-
-    time : timestamp
-        The date/time as found at the client side when an event is
-        created.
+    descriptor : mongoengine.DynamicDocument
+        EventDescriptor foreignkey
```

Comment:
My only point is that it is a significantly different docstring than [this one] (https://github.com/NSLS-II/metadataStore/blob/master/metadataStore/odm_templates.py#L39), [this one] (https://github.com/NSLS-II/metadataStore/blob/master/metadataStore/odm_templates.py#L65) and [this one] (https://github.com/NSLS-II/metadataStore/blob/master/metadataStore/odm_templates.py#L89) for what, looks to me, like the same thing.